doc/nrf/drivers/nrf700x_low_api_common.rst

```diff
@@ -0,0 +1,10 @@
+Common
+------
+
+The common API is used for both normal operation and Radio Test mode.
+
+| Header file: :file:`drivers/wifi/nrf700x/osal/fw_if/umac_if/inc/common/fmac_structs_common.h`
```

Comment:
why do we need this?

Reply:
Just to refer to the header file in the sources, I guess I copied from some existing docs.